src/pgrepr/src/types.rs

```diff
@@ -91,6 +91,9 @@ pub enum Type {
         /// The type of the values in the map.
         value_type: Box<Type>,
     },
+    /// A character type for storing identifiers of no more than 64 characters
```

Comment:
characters or bytes?

Reply:
PostgreSQL defines this as a "Special Character Type" in their docs (https://www.postgresql.org/docs/15/datatype-character.html). So I think character type is probably accurate.

Reply:
Oh, you're talking about the second part
> no more than 64 characters
Yeah, it's bytes not characters. I'll update this comment.

Reply:
Done